Scheduler/observing_blocks_maintel/AOS/LUTs/BLOCK-T186.json

```diff
@@ -0,0 +1,313 @@
+{
+  "name": "BLOCK-T186",
+  "id": "29fb8367-638f-42f8-9ea5-b6c481f2440e",
+  "program": "BLOCK-T186",
+  "constraints": [],
+  "scripts": [
+    {
+      "name": "maintel/track_target.py",
+      "standard": true,
+      "parameters": {
+        "track_azel": {
```

Comment:
Just wondering if what you want is just to point to an (az, el) position and track or look for a bright start. If you don't care about having a bright star, then you are good.

Reply:
That's right, only tracking should suffice. I will rebase and merge. Thanks!